source/controllers/representing-multiple-models.md

```diff
@@ -50,3 +52,39 @@ Now we can use this property in our template:
 
 {{longSongCount}} songs over 30 seconds.
 ```
+
+### Representing Several Models
+
+Multiple models can be returned through `Ember.RSVP.hash`. To continue with the above example, if we also wanted to return the `models/album.js` records then our `model` hook would look like this:
```

Comment:
You mention `Ember.RSVP.hash` but you don't explain why it is necessary for the model hook to work properly. Say something like "we can also return a hash from the model hook, but we need rsvp.hash for it to resolve all of the promises etc"